nirc_ehr/resources/queries/study/pairingSummary.query.xml

```diff
@@ -0,0 +1,25 @@
+<query xmlns="http://labkey.org/data/xml/query">
+<metadata>
+    <tables xmlns="http://labkey.org/data/xml">
+        <table tableName="pairingSummary" tableDbType="TABLE">
+            <tableTitle>Pairing Summary</tableTitle>
+            <javaCustomizer class="org.labkey.nirc_ehr.table.NIRC_EHRCustomizer" />
```

Comment:
See my comments in NIRC_EHRCustomizer. You shouldn't need this.

Reply:
Yup, this file was a copy of pairings, so got copied over